cli/install/install.go

```diff
@@ -865,9 +834,13 @@ func installTarantool(binDir string, incDir string, installCtx InstallCtx,
 	// Check if program is already installed.
 	if !installCtx.Reinstall {
 		log.Infof("Checking existing...")
-		versionExists, err := checkExistingTarantool(versionStr,
-			binDir, incDir, installCtx)
-		if err != nil || versionExists {
+		if util.IsRegularFile(filepath.Join(binDir, versionStr)) &&
+			util.IsDir(filepath.Join(incDir, versionStr)) {
+			log.Infof("%s version of tarantool already exists, updating symlinks...", versionStr)
+			err = changeActiveTarantoolVersion(versionStr, binDir, incDir)
+			if err == nil {
+				log.Infof("Done")
+			}
 			return err
```

Comment:
Seems a more usual way, up to you.
```suggestion
			err = changeActiveTarantoolVersion(versionStr, binDir, incDir)
			if err != nil {
				return err
			}
			log.Infof("Done")
			return nil
```

Reply:
changed

Reply:
Please, change here too.  https://github.com/tarantool/tt/blob/4c74294dc737b746480be5a629bcbd8cca1f229c/cli/install/install.go#L421-L425

Reply:
thx changed